test/contrib/sinatra/tracer_test.rb

```diff
@@ -4,6 +4,7 @@
 class TracerTest < TracerTestBase
   class TracerTestApp < Sinatra::Application
     get '/request' do
+      headers['X-Request-Id'] = request.env['HTTP_X_REQUEST_ID']
```

Comment:
Since current implementation only observes existing headers. So to actually test request_id handling I need to somehow inject the header into response. I'll clarify that in the comments

Reply:
I see; so this is to emulate a Sinatra app that adds a request ID header to its response? Seems okay to me.

Reply:
Yes exactly.